charters/Architecture/charter.md

```diff
@@ -0,0 +1,48 @@
+# DCP Architecture Team
+
+## Description
+
+The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including
+identifying and addressing *cross-cutting* architecture, integration, and operational concerns across DCP
+projects.
+
+The core team is composed of *Technical Leads* representing each technical DCP project group to ensure a diversity of
+skills and views needed for community technical decision-making. 
+
+## In-scope
+
+Software architecture responsibilities
```

Comment:
You could add something here about the architecture team taking responsibility for having an overview and understanding of DCP-wide end to end requirements